### PR TITLE
chore(zappifest): add account param

### PR DESCRIPTION
**Pull request by snooky23**
## Checklist

* [x] PR is scoped to one task
* [ ] Tests are included
* [x] Documentation is included

* [x] This PR is not making any UI change
* [ ] This PR is making a UI change
  * [ ] Screenshot(s) included

## QA :

* required test cases:
* specific apps / plugins to test :

### CODE REVIEW

**Comment by omerweiss** [comment]
@snooky23 , added the whats new doc

File: book/whats_new/20200119_plugiun_accounts.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
+# Plugin accounts
+
+This article will cover the changes we made to the access for plugin developers
+
+## Main Changes
+
+Each plugin has an owning account.
+When submitting a new plugin using zappifest, use the --account with a valid account id:
```
Comment (snooky23) [inline book/whats_new/20200119_plugiun_accounts.md]: --account should be `--account`

File: book/whats_new/20200119_plugiun_accounts.md [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+
+zappifest publish --manifest <path-to-manifest-json-file> --access-token <zapp-access-token> --account <account-id> --new
```
Comment (snooky23) [inline book/whats_new/20200119_plugiun_accounts.md]: zappifest publish --manifest <path-to-manifest-json-file> --access-token <zapp-access-token> --account <account-id> --new

should be 

```
zappifest publish --manifest <path-to-manifest-json-file> --access-token <zapp-access-token> --account <account-id> --new
```

File: book/whats_new/20200119_plugiun_accounts.md [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+
+Exceptions and how to handle them:
+
+You are not permitted to publish, update or delete plugins to this account, please contact support.
+This exception means the user does not have permission to submit, modify or delete plugins to this account.
+
+Contact customer support in order to grant plugin_developer for the account; make sure that the account id is supplied along with the response.
+
+This plugin is associated with another account, please contact support
+This exception means that the plugin is not associated with the account supplied in the --account argument.
```
Comment (snooky23) [inline book/whats_new/20200119_plugiun_accounts.md]: --account should be `--account`

File: book/whats_new/20200119_plugiun_accounts.md [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+Each plugin has an owning account.
+When submitting a new plugin using zappifest, use the --account with a valid account id:
+
+zappifest publish --manifest <path-to-manifest-json-file> --access-token <zapp-access-token> --account <account-id> --new
+
+What is a valid account id?
+
+An account id must comply with the following terms:
+It should be a real account id.
+The zapp user using zappifest must have a plugin_developer for this account.
```
Comment (snooky23) [inline book/whats_new/20200119_plugiun_accounts.md]: plugin_developer should be `plugin_developer`

Please fix it everywhere.

File: book/whats_new/20200119_plugiun_accounts.md [hunk 5]
```diff
@@ -0,0 +1,31 @@
...
+This exception means the user does not have permission to submit, modify or delete plugins to this account.
+
+Contact customer support in order to grant plugin_developer for the account; make sure that the account id is supplied along with the response.
+
+This plugin is associated with another account, please contact support
+This exception means that the plugin is not associated with the account supplied in the --account argument.
+
+Contact a marketplace owner in order to understand which account this plugin is associated with.
+The supporting peer should decide whether to grant permission to the correct account, switch the plugin’s account or reject the request.
+
```
Comment (snooky23) [inline book/whats_new/20200119_plugiun_accounts.md]: Please also add a new section with the below:

We have released a new version of Zappifest v0.62.0 which contains these changes 

From this version, there will be a plugin owner publishing restriction,
Please check the [release notes](https://github.com/applicaster/zappifest/releases/tag/0.62.0) for the complete explanation.

To get the latest Zappifest please run the following:
```
brew update 
brew upgrade zappifest
```

File: book/whats_new/20200119_plugiun_accounts.md (hunk 1 above)
Comment (omerweiss) [reply book/whats_new/20200119_plugiun_accounts.md]: done

**Pull request merged**